analyses/fusion_filtering/01-fusion-filtering.R

```diff
@@ -0,0 +1,259 @@
+library("reshape2")
+library("data.table")
+library("plyr")
+library(ggplot2)
+library(dplyr)
+library(RColorBrewer)
+library(tidyr)
+library(R.utils)
+
+# star fusion
+sf<-read.delim(gzfile(file.path("data","pbta-fusion-starfusion.tsv.gz")),stringsAsFactors=F,,header=T,sep="\t")
+#head(sf)
+sf<-sf[-which(sf$SpanningFragCount-sf$JunctionReadCount >10|sf$JunctionReadCount==0|sf$LargeAnchorSupport == "NO_LDAS"),]
```

Comment:
Thank you so much for going over the code.  Yes the filtering is divided into multiple steps for now but it does make sense to have all the filtering in 1 step. But the filtering to me are three different kinds of filtering 1) gene list filtering  for fusions in genes that belong to (oncogenic,cosmic,tsgs gene list) 2)expression based filtering <1 FPKM needs to be always applied 3) project specific filtering will be the fusion calls in atleast n callers/ atleast n samples per group / atleast n recurrent times per group which I save separately as filtered_fusion_calls and separately run expression filtering (re-run step 2) to remove un-expressed gene fusions.

And for this concern "We can always expect that folks will have RNA-seq expression estimates of some kind because these tools call fusions from RNA-seq data, did I understand that correctly?" Yes we assume that the user will have RNA-seq from the same cohort with the same naming convention to match fusion calls to expression.

Reply:
I'll surely implement the command line arguments as well for easier documentation for parameters